### PR TITLE
Implementation of oemof constraint method and of saving of results

### PR DESCRIPTION
**Pull request by GregorBecker**
- Conversions that were necessary to optimise according to two variables (for example emissions and monetary costs). #33 
- Introduction of an automatically created result folder of the format Scenarioname-date--time which can now be selected for re-view in the GUI. #17

### CODE REVIEW

File: SESMG_EXE.py [hunk 1]
```diff
@@ -160,20 +168,17 @@ def show_graph():
 
 
 def execute_sesmg():
-    """ Excecutes the optimization algorithm """
+    """ 1. Creates the folder where the results would be saved
```
Comment (chrklemm) [inline SESMG_EXE.py:171]: "1. Creates the folder where the results **will** be saved"

File: SESMG_EXE.py [hunk 2]
```diff
@@ -223,6 +228,9 @@ def get_pid():
 
 def show_results():
     """ executes the external program, which executes a plotl.dash app for displaying interactive results."""
+    if save_path.get() == '':
+        raise SystemError('No optimization since the last restart')
```
Comment (chrklemm) [inline SESMG_EXE.py:232]: 'No optimization since the last restart, **please select a result folder!**'

**Review by chrklemm — CHANGES_REQUESTED** [review]
Ein paar kleine Anmerkungen, sonst sieht alles gut aus!

File: SESMG_EXE.py [hunk 3]
```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-
+from datetime import datetime
```
[the remark this answers is not in the document]
Comment (GregorBecker) [reply SESMG_EXE.py:2]: No, because it is a standard python library.

File: SESMG_EXE.py (hunk 1 above)
Comment (GregorBecker) [reply SESMG_EXE.py]: Done.

File: SESMG_EXE.py (hunk 2 above)
Comment (GregorBecker) [reply SESMG_EXE.py]: Done

**Pull request merged**